### PR TITLE
adds a weed cookie

### PR DESCRIPTION
**Pull request by boodaliboo**
the title is self explanatory
apparently since im required to put this pushes everything down in recipe.dm so i can add my cookie in the cookie section
#### Changelog

:cl:  
rscadd: Adds weed cookie
/:cl:

### CODE REVIEW

**Review by Hopekz — CHANGES_REQUESTED** [review]
Undocumented changes in recipes_pastry.dm
Remove them or document them.

File: code/modules/food_and_drinks/food/snacks_pastry.dm [hunk 1]
```diff
@@ -193,6 +193,17 @@
 	tastes = list("cookie" = 1)
 	foodtype = GRAIN | SUGAR
 
+/obj/item/reagent_containers/food/snacks/weedcookie
+	name = "weed cookie"
+	desc = "A cookie mixed with weed, often called a edible."
```
Comment (Ktlwjec1) [inline code/modules/food_and_drinks/food/snacks_pastry.dm:198]: called an edible.

**Comment by SomeguyManperson** [comment]
did you just copy the entire recipies_pastry.dm file from some other codebase and paste it over ours

**Comment by boodaliboo** [comment]
> did you just copy the entire recipies_pastry.dm file from some other codebase and paste it over ours

no?

**Review by JamieD1 — CHANGES_REQUESTED** [review]
Undocumented changes

**Comment by boodaliboo** [comment]
> Undocumented changes

i moved everything up in recipes?

**Comment by JamieD1** [comment]
Can you just redo it without moving everything around? Just add what you need to..
@boodaliboo

**Comment by boodaliboo** [comment]
i forgot

**Pull request closed without merging**